.electron-builder.config.cjs

```diff
@@ -80,6 +80,7 @@ const config = {
   npmRebuild: false,
   beforePack: async context => {
     const DEFAULT_ASSETS = [];
+    const PODMAN_EXTENSION_ASSETS = 'extensions/podman/packages/extension/assets';
```

Comment:
FYI, you can't do refactoring and introduce something new at the same time.
Reading the PR it's not straightforward to know what is related to the support of Arm vs pure code refactoring because you don't want to recopy 'extensions/podman/packages/extension/assets'


first PR = introduce PODMAN_EXTENSION_ASSETS, then adding another item is another